node/node.go

```diff
@@ -734,6 +734,13 @@ func (node *AlgorandFullNode) loadParticipationKeys() error {
 		// Fetch a handle to this database
 		handle, err := node.getExistingPartHandle(filename)
 		if err != nil {
+			if err.Error() == "database is locked" {
```

Comment:
can we type asset here instead of message comparison?

Reply:
no, since the sqlite3 doesn't have a specific type for that error, but I have encapsulated it so it would be cleaner.